### PR TITLE
Add guidance about profile pictures

### PR DESCRIPTION
**Pull request by illicitonion**
## What does this change?

We want our trainees to set up images on their accounts - make this explicit and add image selection guidance. (While still suggesting alternatives for people who can't include images of themselves).

### Common Content?

- `common-content/en/module/induction/accounts/index.md`

### CODE REVIEW

File: common-content/en/module/induction/accounts/index.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ Click on the links to go to the registration pages. Keep your login details for
 We recognise you may need to keep your birth name private. When we say real name, we mean the name you go by. It does not have to match your birth certificate or any other legal document. It has to be consistent and professional.
 {{</note>}}
 
+Every time you create an account which can have a profile picture, you should set one up. Follow these [professional image guidlines](https://www.linkedin.com/business/talent/blog/product-tips/tips-for-taking-professional-linkedin-profile-pictures) when picking an image. Use the same picture for every account.
```
Comment (SallyMcGrath) [inline common-content/en/module/induction/accounts/index.md:21]:
```suggestion
Every time you create an account which can have a profile picture, you should set one up. Follow these [professional image guidelines](https://www.linkedin.com/business/talent/blog/product-tips/tips-for-taking-professional-linkedin-profile-pictures) when picking an image. Use the same picture for every account.
```

**Review by SallyMcGrath — APPROVED** [review]
How did we lose the robohash?

**Comment by illicitonion** [comment]
> How did we lose the robohash?

We didn't lose it, I just moved it up to the top of the page rather than being an after-thought after people have already created accounts :)

**Comment by SallyMcGrath** [comment]
> > How did we lose the robohash?
> 
> We didn't lose it, I just moved it up to the top of the page rather than being an after-thought after people have already created accounts :)

I meant from our collective intelligence really. Robohash has been our profile picture friend for years! (Also a fun Piscine project imo)

**Pull request merged**